src/SnippetProvider/RaygunSnippetProvider.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace SilverStripe\TagManager\SnippetProvider;
+
+use SilverStripe\Core\Environment;
+use SilverStripe\TagManager\SnippetProvider;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms;
+use SilverStripe\Core\Manifest\ModuleResourceLoader;
+
+/**
+ * A snippet provider that lets you add Raygun
+ */
+class RaygunSnippetProvider implements SnippetProvider
+{
+    public function getTitle()
+    {
+        return "Raygun crash reporting (JavaScript front-end)";
+    }
+
+    public function getParamFields()
+    {
+        $imgSrc = ModuleResourceLoader::singleton()->resolveURL('sminnee/tagmanager:client/img/raygun-guide.png');
+
+        return new FieldList(
```

Comment:
You should be able to detect this.

Reply:
Probably the presence of SS_RAYGUN_APP_KEY

Reply:
Right, didn't think about it. I guess we could change the wording if it's installed

Reply:
Done. However, I've left the capability of overriding SS_RAYGUN_APP_KEY in case someone would want to separate backend / frontend issues of the app in two different raygun apps.